tests/decomp/020-cond-flags1/cond-flags1.lst.exp.clean

```diff
@@ -1,9 +1,10 @@
 // Estimated params: [$eax, $ebx]
 void fun_05()
 {
-  $ecx = 0;
   if ($eax_0 != $ebx_0) {
     $ecx = 1;
+  } else {
```

Comment:
Here's the first controversy. My personal writing style favors how it was before. Also, it's arguably "simpler". But that's only surface of it.

Actually, "my writing preference" is because I like to apply trivial optimization ahead of "dumb" compilers (yeah, I remember times when compilers where truly dumb, and even modern are oftentimes "enough simplicity in every wise ..."). But that's just it - optimization. The normal form in my mind is exactly if-else, and I optimize it by the number of jumps. Also, arguably, it takes more effort of human to match the fact that the original assignment gets overwritten in the "if".

So, it's actually the "if-else" rendition is simpler, and a decompiler should optimize for human readability/understandability.

This change thus gets +1 from me.